docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -572,9 +700,29 @@ how your network security groups are configured on AWS.
 Troubleshooting access denied messages can be challenging. A Teleport administrator
 should check to see the following:
 
-- Which roles a user is assigned on "root" when they retrieve their SSH certificate via `tsh login`. You can inspect the retrieved certificate with `tsh status` command on the client-side.
-- Which roles a user is assigned on "leaf" when the role mapping takes place.
-  The role mapping result is reflected in the Teleport audit log. By default,
-  it is stored in `/var/lib/teleport/log` on a *auth* server of a cluster.
-  Check the audit log messages on both clusters to get answers for the
-  questions above.
+- Which roles a user is assigned on the root cluster when they retrieve their SSH
+  certificate via `tsh login`. You can inspect the retrieved certificate with the
+  `tsh status` command on the client-side.
+- Which roles a user is assigned on the leaf cluster when the role mapping takes
+  place. The role mapping result is reflected in the Teleport audit log. By
+  default, it is stored in `/var/lib/teleport/log` on the Auth Server of a
+  cluster. Check the audit log messages on both clusters to get answers for the
+  questions above. 
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+Troubleshooting "access denied" messages can be challenging. A Teleport administrator
```

Comment:
This section is repeated and also appears directly above.  Also, the troubleshooting section as a whole seems mostly relevant to non-cloud installations.

Reply:
There are two separate `TabItem` components here, one which is automatically selected for readers who have chosen "OpenSource" or "Enterprise" in the dropdown menu at the top of the docs site ("Self-Hosted"), and one for readers who have selected "Cloud". Are the correct instructions in each `TabItem`?

Reply:
Yes, when viewing in yarn dev, it looks fine.